packages/c/cmake/xmake.lua

```diff
@@ -106,7 +119,7 @@ package("cmake")
     end)
 
     on_install("@bsd", function (package)
-        os.vrunv("sh", {"./bootstrap", "--prefix=" .. package:installdir()})
+        os.vrunv("sh", {"./bootstrap", "--parallel=6", "--prefix=" .. package:installdir()})
```

Comment:
we can get jobs number  https://github.com/xmake-io/xmake/blob/da3b723be857e3a2fe4e229265caf6246db1f0dd/xmake/modules/package/tools/cmake.lua#L36